apps/design/backend/src/types.ts

```diff
@@ -0,0 +1,81 @@
+// We create new types for precincts that can be split, since the existing
+// election types don't support this. We will likely want to extend the existing
+// types to support it in the future, but doing it separately for now allows us
+// to experiment and learn more first. We'll store these separately in the
```

Comment:
Comment got cut off here

Reply:
Oh, weird - thanks! Must've gotten broken up by the auto-imports  - will fix up